pkg/config/config.go

```diff
@@ -67,3 +67,27 @@ func (cfg *Config) GetFalcoKernelObjPath() string {
 func (cfg *Config) GetEbpfEngineLoaderPath() string {
 	return cfg.data.GetEbpfEngineLoaderPath()
 }
+
+func (cfg *Config) GetUpdateDataPeriod() int {
```

Comment:
I wonder if we should not directly return a duration here, I know you cannot unmarshall a duration, but it's not the client's responsibility to know he should convert int into seconds. WDYT?

Reply:
so do you think it is better that we change the return here time.Second instead?

Reply:
return directly a time.duration that you can use for the ticker